### ISSUE TITLE
`sqlite_query_test` will fail once new hydrofabric versions (`>2.1`) are used

### ISSUE THREAD

**Issue by aaraney** [NOAA-OWP/ngen#864]
`flowpaths` is now `flowlines` in hydrofabrics versions `>2.1`. This test will fail if a newer hf version is used.

https://github.com/NOAA-OWP/ngen/blob/a63cc2621598ea4dc301d6cfac85e21e144872fd/test/geopackage/SQLite_Test.cpp#L47

**Comment by aaraney** [comment]
@program--, would you mind taking a look at this when you get a chance. Seems very low priority atm.

**Comment by program--** [comment]
Ah good catch, maybe this should be changed to test for the existence of `divides`, since that is the same across versions.

**Comment by aaraney** [comment]
Yeah, that is what I was thinking too since `ngen` isn't using `flowpaths` or `flowlines`.
